meetup/tests.py

```diff
@@ -47,65 +91,62 @@ def test_meetup_link_is_unique(self):
     self.assertRaises(IntegrityError, n.save)
 
   def test_getting_all_associated_pizzas(self):
-    meetup= Meetup.objects.create(name="Meeetup1", meetup_link='http://meetup.com/some-meetup')
-    place = meetup.pizza_places.create(name="Pete Zazz")
-    self.assertEquals(place, meetup.pizza_places.all()[0])
+    meetup= Meetup.objects.create(name="Meetup1", meetup_link='http://meetup.com/some-meetup')
+    place = meetup.pizza_places.create(name="Pete Zazz", yelp_link='https://www.yelp.com/biz/prince-st-pizza-new-york')
+    self.assertEquals(place, meetup.pizza_places.first())
 
 class TestMeetupModelValidations(TestCase):
```

Comment:
These tests are getting longer. Each of these classes is a good candidate for a separate file/module.